cmd/webhook/main.go

```diff
@@ -59,12 +64,12 @@ func main() {
 	// set up signals so we handle the first shutdown signal gracefully
 	stopCh := signals.SetupSignalHandler()
 
-	clusterConfig, err := rest.InClusterConfig()
+	cfg, err := clientcmd.BuildConfigFromFlags(*masterURL, *kubeconfig)
```

Comment:
Can we do this for all our binaries?

Reply:
Yes  I will add this into the left activator.